compiler/x/codegen/OMRAheadOfTimeCompile.cpp

```diff
@@ -0,0 +1,140 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2019 IBM Corp. and others
```

Comment:
2020

Reply:
Hi, thank you for the comment, it's fixed now